tests_integration/nativeapp/test_bundle.py

```diff
@@ -7,26 +7,49 @@
 
 from tests.project.fixtures import *
 from tests_integration.test_utils import pushd
+from tests_integration.testing_utils import (
+    assert_that_result_failed_with_message_containing,
+)
 
 USER_NAME = f"user_{uuid.uuid4().hex}"
 TEST_ENV = generate_user_env(USER_NAME)
 
 
-# Tests that we disallow polluting the project source through symlinks
-@pytest.mark.integration
-def test_nativeapp_bundle_does_not_create_files_outside_deploy_root(
-    runner,
-    snowflake_session,
-    temporary_working_directory,
-):
+@pytest.fixture
+def template_setup(runner, temporary_working_directory):
     project_name = "myapp"
     result = runner.invoke_json(
-        ["app", "init", project_name],
+        ["app", "init", project_name],  # Uses default template
+        env=TEST_ENV,
+    )
+    assert result.exit_code == 0
+
+    # Vanilla bundle on the unmodified template
+    result = runner.invoke_json(
+        ["app", "bundle", "--project", project_name],
         env=TEST_ENV,
     )
     assert result.exit_code == 0
+    # The newly created deploy_root is explicitly deleted here, as bundle should take care of it.
+
+    # TODO: replace with assert_dir_snapshot util once available
```

Comment:
nit: maybe not? there's definitely an advantage to making sure on the required bits are present, otherwise the tests can break if we ever add a file to the template or change a README. Not good.